demos/bombs/bombs.h

```diff
@@ -118,7 +118,7 @@ class BombsCanvas : public wxPanel
  * BombsCanvas::x_cell and y_cell
  */
 
-#define BOMBS_FONT wxFont(14, wxROMAN, wxNORMAL, wxNORMAL)
+#define BOMBS_FONT wxFont(14, wxFONTFAMILY_ROMAN, wxFONTSTYLE_NORMAL, wxFONTWEIGHT_NORMAL)
```

Comment:
As long as we modernize this, let's use `wxFontInfo(14).Family(wxFONTFAMILY_ROMAN)` instead which is shorter and arguably more clear.